pages/grid/with-widget-containers-compact.page.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { Header } from "@cloudscape-design/components";
+import { Board, BoardItem } from "../../lib/components";
+import PageLayout from "../app/page-layout";
+import { TestBed } from "../app/test-bed";
+import { ScreenshotArea } from "../screenshot-area";
+import * as i18nStrings from "../shared/i18n";
+import { ItemData } from "../shared/interfaces";
+import { items } from "./with-widget-containers.page";
+
+export default function GridWithWidgetContainerPage() {
+  return (
+    <div className="awsui-compact-mode">
```

Comment:
Should it be a conditional flag on the existing pages?

With the flag, we can write more visual regression tests like `for(const mode of ['compact', 'comfortable'])`

Reply:
I do not think that is necessary. The compact mode only affects two properties and having it on every page for regression is an overkill as of yet. It does also affect the content but that is out of scope to me. That part we can test in the demos instead.

Reply:
Currently there is no test coverage for collision checks in compact mode.

If you do not want to parametrise all pages, you need to add at least one more to test this

Reply:
The collision checks do not depend on the grid parameters. What test case would you recommend to add?

Reply:
I still see a gap in testing which may lead to regressions (not now, but if something changes in the future).

If you insist on not having this test, I can approve, even if it is concerning

Reply:
I do not insist but trying to figure out the right test setup. Running the existing tests twice does not look necessary and also not possible because the move offset values are different for compact. I am thinking of adding one pointer-based and one keyboard-based test to ensure the basic flow works when in compact.